### ISSUE TITLE
Add ability to specify localized yaml documents for multiple locales using $locales

### ISSUE THREAD

**Issue by jeremydw** [grow/pygrow#112]
Currently, a yaml file with multiple localized yaml documents in it is limited to one locale per document via the `$locale` identifier. We should support indicating that the same document is applicable to multiple locales using a new `$locales` identifier.

A sample document:

```
---
foo: bar
---
$locale: de
foo: baz
---
$locales:
- fr
- it
foo: qux
```

Produces four documents: `<default locale>`, `de`, `fr`, and `it`. Due to `$locales`, FR and IT use the same yaml content.

**Comment by uxder** [comment]
+1!

**Comment by jeremydw** [comment]
This is implemented now, and it'll make the next release `0.0.55` which I hope to have ready to go tonight or tomorrow!
